seacrowd/sea_datasets/flores200/flores200.py

```diff
@@ -0,0 +1,487 @@
+    "uig_Arab",
+    "ukr_Cyrl",
+    "umb_Latn",
+    "urd_Arab",
+    "uzn_Latn",
+    "vec_Latn",
+    "vie_Latn",
+    "war_Latn",
+    "wol_Latn",
+    "xho_Latn",
+    "ydd_Hebr",
+    "yor_Latn",
+    "yue_Hant",
+    "zgh_Tfng",
+    "zsm_Latn",
+    "zul_Latn",
+]
+
+_LANGUAGE_NAMES = [lang for lang in _LANGUAGE_NAMES if (lang.split("_")[0] in _LANGUAGES)]
+
+_LICENSE = Licenses.CC_BY_NC_4_0.value
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: "https://github.com/openlanguagedata/flores/releases/download/v2.0-alpha.2/floresp-v2.0-alpha.2.zip",
+}
+
+_SPLITS = ["dev", "devtest"]
+
+_SENTENCES_PATHS = {lang: {split: os.path.join("floresp-v2.0-alpha.2", split, f"{split}.{lang}") for split in _SPLITS} for lang in _LANGUAGE_NAMES}
+
+_METADATA_PATHS = {split: os.path.join("floresp-v2.0-alpha.2", f"metadata_{split}.tsv") for split in _SPLITS}
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+_SUPPORTED_SCHEMA_STRINGS = [f"seacrowd_{str(TASK_TO_SCHEMA[task]).lower()}" for task in _SUPPORTED_TASKS]
+
+_SCHEMAS = [str(TASK_TO_SCHEMA[task]) for task in _SUPPORTED_TASKS]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+@dataclass
+class Flores200SeacrowdConfig(SEACrowdConfig):
+    """BuilderConfig for Nusantara."""
+
+    first_language_name: str = None
+    second_language_name: str = None
+
+
+class Flores200(datasets.GeneratorBasedBuilder):
+    """
+    The creation of FLORES-200 doubles the existing language coverage of FLORES-101.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = []
+
+    first_lang_name = "eng_Latn"
+
+    for second_lang_name in _LANGUAGE_NAMES:
+        if first_lang_name == second_lang_name:
+            continue
+
+        subset_id = f"{_DATASETNAME}_{first_lang_name}_{second_lang_name}"
+
+        BUILDER_CONFIGS.append(
+            Flores200SeacrowdConfig(
+                name=f"{subset_id}_source",
+                version=SOURCE_VERSION,
+                description=f"{_DATASETNAME} source schema",
+                schema="source",
+                subset_id=subset_id,
+                first_language_name=first_lang_name,
+                second_language_name=second_lang_name,
+            )
+        )
+
+        seacrowd_schema_config: list[SEACrowdConfig] = []
+
+        for seacrowd_schema in _SUPPORTED_SCHEMA_STRINGS:
+
+            seacrowd_schema_config.append(
+                Flores200SeacrowdConfig(
+                    name=f"{subset_id}_{seacrowd_schema}",
+                    version=SEACROWD_VERSION,
+                    description=f"{_DATASETNAME} {seacrowd_schema} schema",
+                    schema=f"{seacrowd_schema}",
+                    subset_id=subset_id,
+                    first_language_name=first_lang_name,
+                    second_language_name=second_lang_name,
+                )
+            )
+
+        BUILDER_CONFIGS.extend(seacrowd_schema_config)
```

Comment:
We need to take all the language pairs where one of the language is in the `_LANGUAGES` list